### PR TITLE
Dockerfile: add -Xcheck:jni to jvm args

### PR DESCRIPTION
**Pull request by erizocosmico**
do not merge yet

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,13 +13,20 @@ ENV BBLFSH_HOST bblfshd
...
 ENV BBLFSH_PORT 9432
 
+# jvm variables
+ENV SPARK_JVM_EXTRA_ARGS spark.driver.extraJavaOptions
+ENV JVM_EXTRA_ARGS -Xcheck:jni
+
 USER root
 
 RUN echo "$SPARK_DRIVER_EXTRA_CLASSPATH $SRCD_JAR\n$SPARK_EXECUTOR_EXTRA_CLASSPATH $SRCD_JAR"\
           > /usr/local/spark/conf/spark-defaults.conf
```
Comment (bzz) [inline Dockerfile]: While at this, do you think we could change `>` to `>>` here as well?
That might come handy, incase we decide to distribute some `spark-defaults.conf` one day.

Comment (erizocosmico) [reply Dockerfile]: :+1:

**Comment by erizocosmico** [comment]
We might not need this now, so perhaps we can close it and reopen it if there is some other issue in the future with the C lib.

**Comment by bzz** [comment]
Sounds good!

**Comment by erizocosmico** [comment]
There was no significant change in the output, so not very worth it :(

**Pull request closed without merging**